`azurerm_cognitive_account`: allows `bypass` property for `network_acls` (#28221)
hashicorp/terraform-provider-azurerm: website/docs/r/cognitive_account.html.markdown

A `network_acls` block supports the following:

> * `bypass` - (Optional) Whether to allow trusted Azure Services to access the service. Possible values are `None` and `AzureServices`. Only the `Kind` of `OpenAI` is supported. Defaults to `AzureServices`.
> 

Reviewer: There is no default value for `bypass` set in the schema currently, and if `bypass` can only be used when `kind` is `OpenAI`, then I don't think we should set a default for this property.

It would be good to add a note under the property about which kind is supported rather than having it in the description too.

```suggestion
* `bypass` - (Optional) Whether to allow trusted Azure Services to access the service. Possible values are `None` and `AzureServices`. 

-> **NOTE:** `bypass` can only be set when `kind` is set to `OpenAI `.

```
